### PR TITLE
Added code to suppress intermediate transition animations for pop(to:)

### PR DESCRIPTION
**Pull request by jackrwright**
Hello, I went ahead and created a PR so you can review the changes. If I haven't followed your process correctly, please let me know. Thanks!

### CODE REVIEW

**Comment by jimmy0251** [comment]
I would avoid adding a new state variable. Maybe we can remove multiple paths from the `paths` variable in a single operation?

**Comment by jackrwright** [comment]
Yes, this is much cleaner. I changed the code to modify `paths` in a single operation.

File: Sources/UIPilot/UIPilot.swift [hunk 1]
```diff
@@ -36,7 +36,7 @@ public class UIPilot<T: Equatable>: ObservableObject {
         }
     }
     
-    public func popTo(_ route: T, inclusive: Bool = false) {
+    public func popTo(_ route: T, inclusive: Bool = false, showIntermediateTransitions: Bool = false) {
         logger.log("UIPilot: Popping route \(route).")
```
Comment (jackrwright) [inline Sources/UIPilot/UIPilot.swift:40]: Another question: the way I've implemented this makes it the default behavior for all clients of the library. And changing that behavior requires passing `showIntermediateTransitions: true`. Is that ok? If not, I could make the default `true`. The downside for me would be the need to pass that arg with every call to `popTo()`.

As an alternative, UIPilot.init could accept a default parameter for this.

Comment (jimmy0251) [reply Sources/UIPilot/UIPilot.swift:40]: Thanks for the change Jack, it looks much cleaner now. About flag, I don't think anyone would be interested to show intermediate transitions. If they want, they can always do it themselves with a for loop. Maybe we can make "showIntermediateTransitions: false" the default behavior and can get rid of the flag, what are your thoughts?

On the side note, I'm going to add an API to expose current path of the UIPilot. That will give caller much more control.

Comment (jackrwright) [reply Sources/UIPilot/UIPilot.swift]: I agree that the desire to view the intermediate transitions would be unusual. So, I updated the change to remove the flag and make that behavior the default. Thanks!

**Review by jimmy0251 — APPROVED** [review]

**Comment by jimmy0251** [comment]
Thank you very much for the contribution Jack! This is super helpful in one of our products as well.

**Pull request merged**